docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/index.md

```diff
@@ -8,17 +8,15 @@ hide_title: true
 import Badges from '@site/src/components/Badges';
 import ThemedImage from '@theme/ThemedImage';
 import DocCardList from '@theme/DocCardList';
-
 ```
-:::info
-Please note that this data model is under the Snowplow Personal & Academic License ([SPAL](/docs/contributing/personal-and-academic-license-faq/)).
-:::
 
 :::danger
 The Unified Digital Model is currently in public preview state.
 :::
 
-<Badges badgeType="dbt-package Release" pkg="unified"></Badges>
+<Badges badgeType="dbt-package Release" pkg="unified"></Badges>&nbsp;
+<Badges badgeType="Early Release"></Badges>&nbsp;
```

Comment:
If we're using the Tracker Maintenance Classification badges to refer to data models, we should update some of the wording on this page so that it applies to both trackers and data models: https://deploy-preview-692--snowplow-docs.netlify.app/docs/collecting-data/collecting-from-own-applications/tracker-maintenance-classification/